docs/Conformance Criteria/README-Conformance Criteria V2.md

```diff
@@ -0,0 +1,349 @@
+ </tr>
+
+ <tr>
+   <th style="background-color:#555555">19</th>
+   <th style="background-color:#555555">v2</th>
+   <th style="background-color:#AAAAAA"></th>
+   <th style="background-color:#AAAAAA">x</th>
+   <th></th>
+   <td><b>Matching Zowe CLI Plugin:</b> Provide a Zowe CLI Plugin for the data provider's new profile type that implements the core capabilities required for the new protocol that users can then also use to interact with the protocol outside of the Zowe Explorer extension using Zowe CLI commands.</td>
+ </tr>
+
+ <tr>
+   <th style="background-color:#555555">20</th>
+   <th style="background-color:#555555">v2</th>
+   <th style="background-color:#AAAAAA">x</th>
+   <th style="background-color:#AAAAAA"></th>
+   <th></th>
+   <td><b>Data provider API implementation:</b> Extension fully implements and registers to at least one of the three Zowe Explorer interfaces or alternatively throw exceptions that provide meaningful error messages to the end-user in the 'Error.message' property that will be displayed in a dialog.</td>
+ </tr>
+
+ <tr>
+   <th style="background-color:#555555">21</th>
+   <th style="background-color:#555555">v2</th>
+   <th style="background-color:#AAAAAA"></th>
+   <th style="background-color:#AAAAAA">x</th>
+   <th></th>
+   <td><b>API test suite implementation:</b>  If the extension implements a Zowe Explorer API data provider interface, it should implement a test suite that calls each of the implemented API methods.</td>
+ </tr>
+</table>
+
+### Extension Adding Menus
+
+Criteria for VS Code extensions adding menu and commands to VS Code that utilize Zowe Explorer data or extend Zowe Explorer capabilities.
+
+<table rules="all">
+ <thead>
+ <th style=background-color:#5555AA>Item </th>
+ <th style=background-color:#5555AA>Ver </th>
+ <th style=background-color:#5555AA>Required </th>
+ <th style=background-color:#5555AA>Best Practice </th>
+ <th style=background-color:#5555AA>Conformant </th>
+ <th style=background-color:#5555AA>Criteria </th>
+ </thead>
+
+ <tr>
+   <th style="background-color:#555555">22</th>
+   <th style="background-color:#555555">v2</th>
+   <th style="background-color:#AAAAAA">x</th>
+   <th style="background-color:#AAAAAA"></th>
+   <th></th>
+   <td><b>Command operations: </b> If the extension is adding new commands to Zowe Explorer's tree views, the commands must not replace any existing Zowe Explorer commands.</td>
+ </tr>
+
+  <tr>
+   <th style="background-color:#555555">23</th>
+   <th style="background-color:#555555">v2</th>
+   <th style="background-color:#AAAAAA">x</th>
+   <th style="background-color:#AAAAAA"></th>
+   <th></th>
+   <td><b>Command categories: </b>  If the extension adds to <code>contributes.commands</code> in <code>package.json</code>, the value assigned to the <code>category</code> property contains the extension name and it cannot be "Zowe Explorer".</td>
```

Comment:
did we decide on this being required?  I thought this was discussed to be broken down into 2 where it was best practice to include the extension name but required to not be "Zowe Explorer"?

Reply:
@JillieBeanSim I think it makes sense to break this item down into two as you explained, as this would provide guidance and allow some flexibility for extenders while avoiding misuse of the "Zowe Explorer" category. Perhaps we could word it as follows:
- **Command categories 1:** If the extension adds to `contributes.commands` in `package.json`, the value assigned to the `category` property contains the extension name. (Best practice)
- **Command categories 2:** If the extension assigns values to the `category` property in `contributes.commands` of `package.json`, the values cannot be "Zowe Explorer". (Required if applicable)